gluten-ut/common/src/test/scala/io/glutenproject/utils/BackendTestSettings.scala

```diff
@@ -100,6 +110,22 @@ abstract class BackendTestSettings {
       exclusion.add(ExcludeByPrefix(prefixes: _*))
       this
     }
+    def includeGlutenTestByPrefix(prefixes: String*): SuiteSettings = {
+      inclusion.add(IncludeGlutenTestByPrefix(prefixes: _*))
+      this
+    }
+    def excludeGlutenTestByPrefix(prefixes: String*): SuiteSettings = {
+      exclusion.add(ExcludeGlutenTestByPrefix(prefixes: _*))
+      this
+    }
+    def includeAllGlutenTests(): SuiteSettings = {
+      inclusion.add(IncludeByPrefix(GLUTEN_TEST))
+      this
+    }
+    def excludeAllGlutenTest(): SuiteSettings = {
+      exclusion.add(ExcludeByPrefix(GLUTEN_TEST))
+      this
+    }
```

Comment:
nit, What if using names

`includeByPrefixGluten`
`excludeByPrefixGluten`
`includeAllGluten`
`excludeAllGluten`

Reply:
Since we didn't have a `test` suffix previously when naming the existing methods.

Reply:
But `includeAllGlutenTests` looks fine either, I'll be OK to both.